gitlab/files/gitlab.rb

```diff
@@ -475,7 +503,7 @@
 gitlab_workhorse['listen_network'] = "tcp"
 # gitlab_workhorse['listen_network'] = "unix"
 # gitlab_workhorse['listen_umask'] = 000
-gitlab_workhorse['listen_addr'] = "127.0.0.1:8181"
+gitlab_workhorse['listen_addr'] = "{{ server.workhorse_host }}:{{ server.workhorse_port }}"
```

Comment:
We should introduce standard metadata model for this, so something like:

```
server:
  workhorse:
    bind:
      host: 127.0.0.1
      port: 8181
```

The same for unicorn bellow.